build-tests/rush-redis-cobuild-plugin-integration-test/sandbox/sharded-repo/projects/e/config/rush-project.json

```diff
@@ -1,10 +1,16 @@
 {
+  "$schema": "../../../../../../../libraries/rush-lib/src/schemas/rush-project.schema.json",
   "operationSettings": [
     {
       "operationName": "_phase:build",
       "outputFolderNames": ["dist"],
+      "allowCobuildOrchestration": true,
+      "disableBuildCacheForOperation": true,
       "sharding": {
-        "count": 75
+        "count": 75,
+        "shardOperationSettings": {
+          "allowCobuildOrchestration": true
```

Comment:
This name is confusing. Doesn't orchestration always happen? Maybe something like `"allowCobuildWithoutCache"`?

Reply:
Updated in [c0fd113](https://github.com/microsoft/rushstack/pull/4871/commits/c0fd113fc40764c6525cec84d7ac06eeae836b68)